tools/PublishModules.ps1

```diff
@@ -336,10 +436,17 @@ if ($repo -ne $null) {
     Register-PSRepository -Name $tempRepoName -SourceLocation $tempRepoPath -PublishLocation $tempRepoPath -InstallationPolicy Trusted -PackageManagementProvider NuGet
 }
 
-$env:PSModulePath="$env:PSModulePath;$tempRepoPath"
+
+
+        $sourceModulePath = "$packageFolder\ResourceManager\AzureResourceManager"
+
+$sourceModulePath
```

Comment:
Do you want to print this out? Seems like this and line 445 should be removed.

Reply:
This might be a leftover from the debugging on Friday... do we want this left in?